test/scale-infra/README.md

```diff
@@ -0,0 +1,83 @@
+A scalable Kubernetes-based testbed for KubeStellar Performance Tests
+---------------------------------------------------------------------
+
+<img src="images/ks-scale-test-infra.jpg" width="60%" height="60%" title="ks-scale-test-infra">
+
+### Prerequisites
+- Ansible, with 
+  - the `amazon.aws` collection installed (see, e.g., https://docs.ansible.com/ansible/latest/collections/amazon/aws/ec2_module.html), and
+  - [PyPI package boto3](https://pypi.org/project/boto/) installed for the Python interpreter that Ansible uses on your machine;
+- AWS CLI;
+- AWS access key ID and AWS secret access key (to configure AWS CLI [see](https://docs.aws.amazon.com/cli/v1/userguide/cli-configure-files.html#cli-configure-files-methods))
+- An SSH "keypair" registered with EC2
+
+### Overview
+
+The testbed consists of (from bottom to top):
+- Networking and storage resources from AWS;
+- AWS EC2 instances;
+- Kubernetes cluster(s);
+- Workload(s): Kubestellar Core and WEC components 
+
+
+### Quick Start Deployment Instructions:
+
+1. Deploy the KS control plane hosting infra: 
+
+    ```
+    ./deploy_ks_cp_infra.sh --region us-east-2 --k8s_num_masters 3 --k8s_num_workers 3 --instances_type t2.xlarge   --aws_key_name mykey  --arch x86_64 --ks_release 0.25.0
+    ```
+
+    The above command creates the required AWS infrastructure including a VPC, security groups and EC2 instances. Then, it creates a Kubernetes cluster deployed using Kubeadm. Lastly, it deploys the KubeStellar core components. You can use the flag `--ks_release` to specify the KubeStellar release.
+
+
+2. Create the WEC hosting instances:
+
+    ```
+    ./deploy_wec_infra.sh --region us-east-2 --num_hosts 1 --instance_type t2.2xlarge --aws_key_name  mykey  --arch x86_64
+    ```
+
+    Use the flag `--num_hosts` to specify the number of ec2 instances to be created. 
+
+3. Create the WEC kind clusters:
+
+    a) Update the inventory of the WEC instances to include the IP address of one of the master nodes of the K8s cluster created in step-1 above:
+
+    First, find the required master's IP address info in the following file: `.data/hosts_core`.
+
+    Next, use the following command to see the content of the WEC ansible inventory:
+    ```bash
+    cat .data/hosts_wec
+    ```
+
+    Sample output: 
+    ```console
+    [masters]
+    <add master node info here!>
+
+    [add_workers]
+    worker1 ansible_host=192.168.56.1
+    ```
+
+    Lastly, edit the contents of the file `.data/hosts_wec` to include the IP address of a master node:
+    ```bash
+    vi .data/hosts_wec
+    ```
+
+    b) Create Kind cluster WECs and connect to KS Core cluster
+
+    ```
+    ansible-playbook -i .data/hosts_wec deploy_ks_wec.yaml --ssh-common-args='-o StrictHostKeyChecking=no' -e 'num_wecs=1  wec_name_prefix=location'
+    ```
+
+    Use the input paramater `num_wecs` to specify the number of kind clusters to be created. 
```

Comment:
Is this number spread evenly among the WEC Hosting Instances, or repeated in each one?

Reply:
It's repeated in each WEC Hosting Instance. Hence, I updated that sentence to make it a bit clear as following: 

```
 Use the input paramater `num_wecs` to specify the number of kind clusters to be created for each WEC Hosting Instances ...
```